scripts/train_tokenizer.py

```diff
@@ -0,0 +1,53 @@
+import argparse
+from pathlib import Path
+
+from tokenizers.implementations import BertWordPieceTokenizer
+
+from utils.utils import load_json
```

Comment:
`from src.utils.utils ...`
Btw, maybe rename `utils.py` to something like `common.py`. Just to omit this repetition in import